pkg/dashboard/api/types.go

```diff
@@ -120,6 +121,7 @@ type Alert struct {
 	Type           string `json:"type"`
 	Name           string `json:"name"`
 	Text           string `json:"text"`
+	DeploymentUrl  string `json:"deploymentUrl"`
```

Comment:
This is a calculated value. With relevancy only in NotifManager. Calculate it there.